app/src/PageController.php

```diff
@@ -20,10 +18,10 @@ protected function init()
         Requirements::themedCSS('form');
         Requirements::themedCSS('typography');
 
-        Requirements::css('mysite/css/demo.css');
-        Requirements::css('mysite/css/moduleSupport.css');
-        Requirements::javascript('silverstripe/admin:thirdparty/jquery/jquery.js');
-        Requirements::javascript('mysite/javascript/googleanalytics.js');
+        Requirements::css('app/css/demo.css');
+        Requirements::css('app/css/moduleSupport.css');
+        Requirements::javascript('//code.jquery.com/jquery-3.6.4.min.js');
```

Comment:
jQuery is only used to wire some GA events in the front end. I had a glance to make sure the thing wasn't obviously broken.

It still works just fine. Didn't look too closely because I don't think we need to worry analytics on a project we are looking at nuking in the next few month anyway.